packages/flutter/lib/src/widgets/focus_scope.dart

```diff
@@ -901,3 +942,46 @@ class _FocusMarker extends InheritedNotifier<FocusNode> {
         assert(child != null),
         super(key: key, notifier: node, child: child);
 }
+
+/// A widget that controls whether or not the descendants of this widget are
+/// focusable.
+///
+/// Does not affect the value of [canRequestFocus] on the descendants.
+///
+/// See also:
+///  - [Focus], a widget for adding and managing a [FocusNode] in the widget tree.
```

Comment:
nit: blank line after "See also:" and for consistency * instead of - in the enumeration.

Reply:
Gosh, and I did a whole cleanup PR of those as penance a while ago.  You'd think I'd remember! Fixed!